README.md

```diff
@@ -1,24 +1,1069 @@
+```
+
+Response Body
+```
+{
+  id: 539,
+  name: "Item Sunt Saepe",
+  description: "Quibusdam aut qui. Sint repudiandae et dolor quo. Et nihil qui beatae est ipsam quo corrupti. Voluptatibus ut pariatur dolorem nostrum temporibus magnam enim. Quia nisi voluptate aperiam vel et.",
+  merchant_id: 26,
+  unit_price: "136.35"
+}
+```
+
+Response Code
+```
+200
+```
+
+#### Transaction Endpoints
+**Transaction Index**
+
+Request URL
+```
+http://localhost:3000/api/v1/transactions
+```
+
+Response Body
+```
+[
+  {
+    credit_card_number: "4654405418249632",
+    id: 1,
+    invoice_id: 1,
+    result: "success"
+  },
+  {
+    credit_card_number: "4580251236515201",
+    id: 2,
+    invoice_id: 2,
+    result: "success"
+  },
+  {...},
+]  
+```
+
+Response Code
+```
+200
+```
+**Transaction Show**
+
+Request URL
+```
+http://localhost:3000/api/v1/transactions/:id
+```
+
+Response Body
+```
+{
+  credit_card_number: "4654405418249632",
+  id: 1,
+  invoice_id: 1,
+  result: "success"
+}  
+```
+
+Response Code
+```
+200
+```
+
+##### Transaction Relationship Endpoints
+
+**Invoice**
+
+Request URL
+```
+http://localhost:3000/api/v1/transactions/:id/invoice
+```
+
+Response Body
+```
+{
+  id: 1,
+  customer_id: 1,
+  merchant_id: 26,
+  status: "shipped"
+}
+```
+
+Response Code
+```
+200
+```
+
+#### Customer Endpoints
+**Customer Index**
+
+Request URL
+```
+http://localhost:3000/api/v1/customers
+```
+
+Response Body
+```
+[
+  {
+    first_name: "Joey",
+    id: 1,
+    last_name: "Ondricka"
+  },
+  {
+    first_name: "Cecelia",
+    id: 2,
+    last_name: "Osinski"
+  },
+  {...},
+]  
+```
+
+Response Code
+```
+200
+```
+
+**Customer Show**
+
+Request URL
+```
+http://localhost:3000/api/v1/customers/:id
+```
+
+Response Body
+```
+{
+  first_name: "Joey",
+  id: 1,
+  last_name: "Ondricka"
+}
+```
+
+Response Code
+```
+200
+```
+
+##### Customer Relationship Endpoints
+**Invoices**
+
+Request URL
+```
+http://localhost:3000/api/v1/customers/:id/invoices
+```
+
+Response Body
+```
+[
+  {
+    id: 8,
+    customer_id: 1,
+    merchant_id: 38,
+    status: "shipped"
+  },
+  {
+    id: 7,
+    customer_id: 1,
+    merchant_id: 44,
+    status: "shipped"
+  },
+  {...},
+]  
+```
+
+Response Code
+```
+200
+```
+
+**Transactions**
+
+Request URL
+```
+http://localhost:3000/api/v1/customers/:id/transactions
+```
+
+Response Body
+```
+[
+  {
+    credit_card_number: "4801647818676136",
+    id: 7,
+    invoice_id: 8,
+    result: "success"
+  },
+  {
+    credit_card_number: "4203696133194408",
+    id: 6,
+    invoice_id: 7,
+    result: "success"
+  },
+  {...},
+]  
+```
+
+Response Code
+```
+200
+```
+
+##### Customer Business Intelligence Endpoints
+
+##### *Single Customer*
+
+
+**Merchant with Whom Customer Has Completed Most Transactions**
+
+Request URL
+```
+http://localhost:3000/api/v1/customers/:id/favorite_merchant
+```
+
+Response Body
+```
+{
+  id: 26,
+  name: "Balistreri, Schaefer and Kshlerin"
+}
+```
+
+Response Code
```

Comment:
One thought since the endpoint outline takes up quite a bit of space... What do you think about putting the contribute, schema, contributors, and versions above it?

Reply:
Sure, I'll make that change now